src/value.stub.php

```diff
@@ -7,7 +7,7 @@
 
 
 class Value {
-	public function __construct(mixed $value) {}
+	public function __construct(mixed $value = NULL) {}
```

Comment:
I don't really understand this pieve of code

```
	if (php_componere_parse_parameters("|z", &value) != SUCCESS) {
		php_componere_wrong_parameters("value expected");
```

Is it a bad error message, or value is really mandatory ?

Reply:
Looks like the result of a mass refactoring when introducing parse_parameters/wrong_parameters ... feel free to fix that.

Reply:
Really optional (lot of new Value() in test suite), so I only change the exception message to `"only optional value expected"`